### PR TITLE
#168362842 View all mentors 

### PR DESCRIPTION
**Pull request by sharonandisi**
What does this PR do?
This is a feature that allows  user to view all mentors
Description of Task to be completed?
As a user I should be able to view all mentors
How should this be manually tested?
By cloning this repo and setting it up and run npm start-dev
What are the relevant pivotal tracker stories?
[#168362842](https://www.pivotaltracker.com/n/projects/2379605)
Screenshot from 2019-09-12 12-47-59

### CODE REVIEW

File: server/controllers/mentorController.js [hunk 1]
```diff
@@ -1,7 +1,23 @@
-import { findOne } from '../models/userModel';
+import { findMentor, findOne } from '../models/userModel';
 import response from '../helpers/responseHelper';
 import messageHelper from '../helpers/messageHelper';
 
+async function fetchAllMentors(req, res) {
+    try {
+        const mentor = await findMentor();
+        const mentors = mentor.map(({ id: mentorid, firstname, lastname, email, address, occupation, bio, expertise, mentorstatus, isAdmin, }) => ({
+            mentorid, firstname, lastname, email, address, occupation, bio, expertise, mentorstatus, isAdmin,
```
Comment (ekumamait) [inline server/controllers/mentorController.js:9]: if you can, do not return the mentorstatus

**Review by ekumamait — COMMENTED** [review]
Can you raise the coverage?

**Comment by sharonandisi** [comment]
> Can you raise the coverage?

File: server/controllers/mentorController.js (hunk 1 above)
Comment (sharonandisi) [reply server/controllers/mentorController.js:9]: alright. Can I fix this on the next PR ? and yes. I can raise the coverage

**Pull request merged**